bfh/fields.py

```diff
@@ -33,6 +33,10 @@ class so you can implement your own if you want.
 ]
 
 
+def get_default(value):
+    return value() if callable(value) else value
```

Comment:
This feels Java-esque. Do you need separate `get_default` and `default` getter? Could you just put the evaluation of the callable in `default`? Or go the other way, `get_default` can access self._default directly.